src/node_i18n.cc

```diff
@@ -489,8 +492,11 @@ static void ToUnicode(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(args.Length(), 1);
   CHECK(args[0]->IsString());
   Utf8Value val(env->isolate(), args[0]);
+  // optional arg
+  bool lenient = args[1].As<Boolean>()->Value();
```

Comment:
I didn't update the check for argument length, since (as the comment is trying to say) it is an optional argument, so that existing usage of `toUnicode(str)` would still work. V8 automatically returns an `Undefined` for out-of-range `args[]` dereference.

Wasn't aware of `BooleanValue()`. Will use that instead.